rest/test_results_routes_test.go

```diff
@@ -316,6 +318,96 @@ func (s *TestResultsHandlerSuite) TestTestResultGetByTestNameHandlerCtxErr() {
 	s.Equal(http.StatusInternalServerError, resp.Status())
 }
 
+func (s *TestResultsHandlerSuite) TestTestResultsGetFailedSampleFound() {
+	rh := s.rh["failed_tests_sample"]
```

Comment:
In these tests, it seems like you always type assert that it's a testResultsGetFailedSampleHandler. It seems like you could just type assert it here instead of doing repeated type asserts.